apps/07jquery-tic-tac-toe/index.html

```diff
@@ -23,7 +23,7 @@
   </div>
   <div id="announce-winner"></div>
   <button id="clear">Clear Board</button>
-  <script type="text/javascript" src="../vendor/jquery.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.js"></script>
```

Comment:
@tjefferson08 When i cd into the tic tac toe directory, and run `http-server`, the vendor version doesn't work.

I may have done something wrong. Would like you to comment on this.